docs/reference/indices/flush.asciidoc

```diff
@@ -24,21 +27,21 @@ The flush API accepts the following request parameters:
 
 [horizontal]
 `wait_if_ongoing`::  If set to `true` the flush operation will block until the
-flush can be executed if another flush operation is already executing.
-The default is `false` and will cause an exception to be thrown on
-the shard level if another flush operation is already running.
+flush can be executed if another flush operation is already executing. The
```

Comment:
the main point is here that the command waits until a flush has happened and all operations indexed before the command was called are guaranteed to be flushed. I'm tempting to also say we need to change the default, but that's a different discussion :)

Reply:
Ok. This wasn't spelled out before as far as I can see, but it's a good point to make. See https://github.com/elastic/elasticsearch/pull/46245/files#diff-31f3afe6b50f556a95a396324be535e5R22-R24.

The default is `true`, fixed by Nhat recently.